Apps/WebClient/src/ClientApp/src/stores/specialAuthorityRequest.ts

```diff
@@ -135,10 +141,11 @@ export const useSpecialAuthorityRequestStore = defineStore(
                     if (result.resultStatus === ResultType.Error) {
                         throw result.resultError;
                     }
-                    EventTracker.loadData(
-                        EntryType.SpecialAuthorityRequest,
-                        result.resourcePayload.length
-                    );
+                    trackingService.trackEvent({
+                        action: Action.Load,
+                        text: Text.Data,
+                        dataset: Dataset.SpecialAuthorityRequests,
+                    });
```

Comment:
`EventTracker.loadData()` would only track a load event if at least 1 record was returned for the dataset. That logic may need to be included here to preserve the same behaviour.

(Applies to all dataset stores.)

Reply:
Changed.